xtask/src/main.rs

```diff
@@ -35,13 +36,15 @@ enum Cli {
     /// Bump the version of the specified package(s).
     BumpVersion(BumpVersionArgs),
     /// Format all packages in the workspace with rustfmt
+    #[clap(alias = "format-packages")]
```

Comment:
We also have [`cargo xfmt`](https://github.com/esp-rs/esp-hal/blob/c19c7fcf01bac8eb82be38a8ddaf603f191a61a5/.cargo/config.toml#L3) if you want to save a few keystrokes :D

Reply:
It's more just habit than anything at this point 😁 Thanks for pointing that out, though.